task/task.json

```diff
@@ -75,6 +75,15 @@
       "visibleRule": "AuthScheme = sqlServerAuthentication",
       "helpMarkDown": "Provide the SQL login to connect to the SQL Server. The option is only available if SQL Server Authentication mode has been selected."
     },
+    {
+      "groupName":"server",
+      "name": "InitialCatalog",
+      "type": "string",
+      "label": "SSISDB Catalog Name",
+      "required": false,
```

Comment:
I think this should be a required param and default it to master, as mentioned above.